PROJECT-CLEARWATER-MIB

```diff
@@ -5050,6 +5050,80 @@ chronosInvalidTimersProcessedCount OBJECT-TYPE
     DESCRIPTION "The total number of invalid timers processed over the period."
     ::= { chronosInvalidTimersProcessedEntry 2 }
 
+chronosAllTimersTable OBJECT-TYPE

```

Comment:
You need to also add these to the ChronosGroup - see https://github.com/Metaswitch/clearwater-snmp-handlers/blob/master/PROJECT-CLEARWATER-MIB#L5161